### ISSUE TITLE
S3449 FN: Remove unsigned right-shift operator test cases (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6402]
### Description

FN for S3449 (ShiftDynamicNotInteger).

Using C#11 unsigned right-shift operator, the rule is not raising an issue.

### Repro steps

```csharp
int d = 5;
var x = d >>> null; // FN
int? y = 5;
y >>>= null; // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
